src/components/Image/index.js

```diff
@@ -130,17 +130,21 @@ export default class Image extends Component {
       accessible,
       children,
       defaultSource,
-      resizeMode,
       source,
-      style,
       testID
     } = this.props
+    const style = flattenStyle(this.props.style)
 
     const isLoaded = this.state.status === STATUS_LOADED
     const defaultImage = defaultSource.uri || null
     const displayImage = !isLoaded ? defaultImage : source.uri
     const backgroundImage = displayImage ? `url("${displayImage}")` : null
 
+    const resizeMode = this.props.resizeMode || (style || {}).resizeMode || 'cover'
```

Comment:
Rather than using the string `cover`, we can use `ImageResizeMode.cover`.

We should be able to avoid the potential creation of a new object in `render`, for example by including a default value for `style` in `defaultProps`. That would also avoid the need for the condition around `delete`.